.github/workflows/main_deploy.yaml

```diff
@@ -3,6 +3,10 @@ on:
   schedule:
     - cron:  '0 */4 * * *'
   push:
+    paths:
+      - '**.js'
+      - '!**.md'
+      - '!**.yaml'
```

Comment:
We want it to run if changes are made in `readme-profiles/*.md` Also if changes are made to `contributors.json`. Also anything inside `src` folder

Reply:
@subeshb1 I have made the changes. Please check.